README.md

```diff
@@ -24,15 +24,15 @@ Setup
 
 1. Add JCenter repository to your Maven or Gradle build:
 	1. For Maven, please read instructions at: https://bintray.com/bintray/jcenter
-	2. For Gradle, add `jcenter()` to your `repositories { ... }`
+	2. For Gradle, add `mavenCentral()` to your `repositories { ... }`
 2. Add the dependency
     1. For Maven:
         ```xml
             <dependency>
 
         
 
-                <groupId>com.cnp</groupId>
+                <groupId>io.github.vantiv</groupId>
                 <artifactId>cnp-sdk-for-java</artifactId>
                 <version>12.0.3</version>
```

Comment:
should this version say 12.19.1 as well?

Reply:
Fixed

> Update needed for changelog
Fixed